### PR TITLE
test: add unit test case for func ParseEnv

### PR DESCRIPTION
**Pull request by jahentao**
Signed-off-by: jahentao <jahentao@qq.com>

### Ⅰ. Describe what this PR did

test: add unit test case for func ParseEnv

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)

added

### Ⅳ. Describe how to verify it

```
go test -v -timeout 30s github.com/alibaba/pouch/apis/opts -run "^(TestParseEnv)$"
```

### CODE REVIEW

**Comment by jahentao** [comment]
How can i ask for a review?

**Comment by houstar** [comment]
@jahentao  I'd like to use `git blame` to see who is the originally wrote the code. then ask the maintainer to assign this PR to @hime review. :-)

File: apis/opts/env_test.go [hunk 1]
```diff
@@ -15,9 +15,13 @@ func TestParseEnv(t *testing.T) {
...
-		// TODO: Add test cases.
-		{name: "test1", args: args{env: []string{"foo=bar"}}, want: map[string]string{"foo": "bar"}, wantErr: false},
-		{name: "test2", args: args{env: []string{"ErrorInfo"}}, want: nil, wantErr: true},
+		{"test single env", args{env: []string{"foo=bar"}}, map[string]string{"foo": "bar"}, false},
+		{"test error env format", args{env: []string{"ErrorInfo"}}, nil, true},
+		{"test multiple envs", args{env: []string{"foo=bar", "A=a"}}, map[string]string{"foo": "bar", "A": "a"}, false},
+		{"test multiple '=' envs", args{env: []string{"A=1=2"}}, map[string]string{"A": "1=2"}, false},
+		{"test nil env", args{env: nil}, map[string]string{}, false}, // empty map
+		{"test empty env", args{env: []string{""}}, nil, true},
+		{"test empty blank", args{env: []string{"  "}}, nil, true},
```
[the remark this answers is not in the document]
Comment (jahentao) [reply apis/opts/env_test.go]: Thanks for your reviewing. Maybe it not sufficiently test before :-), such as nil and empty env situation, and i need a easy entry point for my first contributing. I will do better later.

[the remark this answers is not in the document]
Comment (ZYecho) [reply apis/opts/env_test.go]: Yep, you can simply make another commit for this pr.

[the remark this answers is not in the document]
Comment (allencloud) [reply apis/opts/env_test.go]: Thanks a lot for adding more test cases. @jahentao 
The code quality is quite important for us.

**Comment by allencloud** [comment]
LGTM

**Pull request merged**